packages/infra/lib/ihe-gateway-v2-stack.ts

```diff
@@ -38,6 +42,53 @@ export class IHEGatewayV2LambdasNestedStack extends NestedStack {
     patientDiscoveryLambda.grantInvoke(props.apiService.taskDefinition.taskRole);
     documentQueryLambda.grantInvoke(props.apiService.taskDefinition.taskRole);
     documentRetrievalLambda.grantInvoke(props.apiService.taskDefinition.taskRole);
+
+    this.addMetricFiltersAndAlarms(patientDiscoveryLambda, "PatientDiscoveryLambda", props, [
+      { filterPattern: "Aborted Error is present in response", threshold: 100 },
+      { filterPattern: "Failure Sending SAML Request", threshold: 100 },
```

Comment:
we have 115/1500 endpoints erroring in prod, split between the two error types: 

aborted errors (inline soap errors) 
http errors (400s, 500s, etc) 

100 allotted for each for now  should be a good buffer.